docs/setup-cluster/workspaces.rst

```diff
@@ -35,9 +35,9 @@ individuals or teams using Determined. This is recommended for larger teams.
 WebUI
 =====
 
-After logging in, the default landing page for the WebUI is **Uncategorized Experiments List**. This
-is where experiments that do not have a configuration file ``workspace`` and ``project`` are put,
-including experiments created before the workspaces feature was released.
+After logging in, the default landing page for the WebUI is the dashboard. This is where experiments
```

Comment:
This description is incorrect. The dashboard shows the most recently submitted jobs and recently viewed projects.

Reply:
thanks. resolved by rewriting the paragraph.